### PR TITLE
Reduce epoch duration from 2 weeks to 2 days

### PR DESCRIPTION
**Pull request by mvines**
2 weeks is a really long time to wait for stake warm up/cool down.

### CODE REVIEW

**Comment by mvines** [comment]
Oh good, a test failed!

**Comment by mvines** [comment]
Added reviewers.  I'd like to get at least two approvals before landing this

File: runtime/src/bank.rs [hunk 1]
```diff
@@ -2878,7 +2878,7 @@ mod tests {
 
     #[test]
     #[allow(clippy::cognitive_complexity)]
-    fn test_rent() {
+    fn test_rent_complex() {
```
Comment (mvines) [inline runtime/src/bank.rs]: This test is a mess 🙈

**Comment by carllin** [comment]
@aeyakovenko seems like a straightforward change. Not sure what the implications are for rent, so soft +1.

**Comment by mvines** [comment]
The rent test churn was just because that test makes hard coded assumptions about the epoch duration as it calculates the amount of rent to expect.   I removed some of that hard coding here, but it's a still a super fragile test

**Pull request merged**